packages/paste-core/components/disclosure/src/Disclosure.tsx

```diff
@@ -16,12 +16,20 @@ export interface DisclosureContextProps {
 
 export const DisclosureContext = React.createContext<DisclosureContextProps>({} as any);
```

Comment:
You aughta be able to type this as DisclosurePrimitveState I think?

Reply:
Not sure I'm following this one. Something like:
`export const DisclosureContext = React.createContext<DisclosureContextProps>({} as DisclosurePrimitveState);`?

Reply:
Yeah, I don't know about setting `any` here. Maybe `Partial<DisclosurePrimitveState>` if `DisclosurePrimitveState` doesn't work?

Reply:
@TheSisb changing to that partial gets me this error: https://github.com/typescript-eslint/typescript-eslint/blob/v1.13.0/packages/eslint-plugin/docs/rules/no-object-literal-type-assertion.md

Looks like `any` might be the answer.

Reply:
This might have been one of those things where the typings from Reakit were a little jank too. Partial might do a decent enough job